### PR TITLE
Phase2 - Milestone 1&2 Mapping

### PR DESCRIPTION
**Pull request by shashanknarang27**

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs [hunk 1]
```diff
@@ -192,7 +233,12 @@ public async Task CheckOnePatient(string templateName, string samplePath, DataTy
         {
             var result = await ConvertData(templateName, samplePath, dataType);
             var patients = result.SelectTokens("$.entry[?(@.resource.resourceType == 'Patient')].resource.id");
-            Assert.Equal(1, patients?.Count());
+            if (templateName == "ADT_A40" || templateName == "ADT_A41" ) {
```
Comment (BoyaWu10) [inline src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs]: Just want to double confirm: are we going to convert these two types of messages to FHIR without a Patient resource?

Comment (shashanknarang27) [reply src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs]: Yes @BoyaWu10 , we are not creating the Patient resource in ADT_A40 and ADT_A41. We are creating the Linkage resource in these messages.

**Comment by BoyaWu10** [comment]
Hi @shashanknarang27, could you please rebase the base branch `shashanknarang27:xoriant/m1` to latest dotliquid branch?

**Comment by shashanknarang27** [comment]
> Hi @shashanknarang27, could you please rebase the base branch `shashanknarang27:xoriant/m1` to latest dotliquid branch?

@BoyaWu10 : We have rebased the base branch `shashanknarang27:xoriant/m1` to latest dotliquid branch

**Comment by yankunhuang-pku** [comment]
Hi @shashanknarang27, the pipeline was fixed. Could you please try to rebase this branch to latest dotliquid branch?

**Comment by shashanknarang27** [comment]
> Hi @shashanknarang27, the unit test for checking the number of default templates was fixed (#360). Could you please try to rebase this branch to latest dotliquid branch?

@yankunhuang-pku : We have rebased the branch to latest dotliquid branch

**Comment by yankunhuang-pku** [comment]
> > Hi @shashanknarang27, the unit test for checking the number of default templates was fixed (#360). Could you please try to rebase this branch to latest dotliquid branch?
> 
> @yankunhuang-pku : We have rebased the branch to latest dotliquid branch

I think we can change the target branch from the branch `hl7v2templates/phase2` to `dotliquid` now.

**Comment by shashanknarang27** [comment]
> > > Hi @shashanknarang27, the unit test for checking the number of default templates was fixed (#360). Could you please try to rebase this branch to latest dotliquid branch?
> > 
> > 
> > @yankunhuang-pku : We have rebased the branch to latest dotliquid branch
> 
> I think we can change the target branch from the branch `hl7v2templates/phase2` to `dotliquid` now. @shashanknarang27 Could you resolve the conflicts?

@yankunhuang-pku : We have resolved the conflicts

**Review by yankunhuang-pku — APPROVED** [review]

File: docs/Filters-and-Tags.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ By default, Liquid provides a set of [standard filters](https://github.com/Shopi
 Besides these filters, FHIR Converter also provides some other filters that are useful in conversion, which are listed below.
 If these filters do not meet your needs, you can also write your own filters.
 
-### Hl7 v2 specific filters
+### Hl7v2 specific filters
```
Comment (BoyaWu10) [inline docs/Filters-and-Tags.md:11]: The changes seem already exist in current `dotliquid` branch. Could you please double check the commit history of the target branch?

File: release.yml [hunk 3]
```diff
@@ -9,6 +9,7 @@ pr:
 - dotliquid
 - main
 - templates/*
+- hl7v2templates/*
```
Comment (BoyaWu10) [inline release.yml:12]: Do we need to make this change into `dotliquid`?

File: docs/Filters-and-Tags.md (hunk 2 above)
Comment (yankunhuang-pku) [reply docs/Filters-and-Tags.md:11]: Yes, it seems you merged the main branch? @shashanknarang27 
image

Comment (shashanknarang27) [reply docs/Filters-and-Tags.md:11]: @yankunhuang-pku : Before pushing the new code for Phase 2 mapping , we have fetch and merge the main branch in Xoriant/m1 and then add the new mapping code in xoriant/m1.

File: release.yml (hunk 3 above)
Comment (shashanknarang27) [reply release.yml:12]: Ok, I will remove this and push the changes

File: docs/Filters-and-Tags.md (hunk 2 above)
Comment (shashanknarang27) [reply docs/Filters-and-Tags.md:11]: @BoyaWu10 and @yankunhuang-pku : I will push the changes for the docs/Filters-and-Tags.md file

**Comment by shashanknarang27** [comment]
@BoyaWu10 and @yankunhuang-pku : I have pushed the changes. Please review the same.

**Review by BoyaWu10 — APPROVED** [review]

**Comment by BoyaWu10** [comment]
> @BoyaWu10 and @yankunhuang-pku : I have pushed the changes. Please review the same.

 Thanks @shashanknarang27! The PR looks good to me.

**Pull request merged**